### PR TITLE
[Feature][Zeta] Zeta support print metrics information

### PR DESCRIPTION
**Pull request by TyrantLucifer**
## Purpose of this pull request

Zeta support print metrics information

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by TyrantLucifer** [comment]
dc8f3140a1dc0c0c7598d23c3bbfc96

0e2f6feb6f83ce06e00b1536c42432c

**Comment by TyrantLucifer** [comment]
image

**Comment by ic4y** [comment]
> image

Is `Last Run Time` called `Last Print Time` or something else?

**Comment by TyrantLucifer** [comment]
> > image
> 
> Is `Last Run Time` called `Last Print Time` or something else?

Good idea. How about `Last Statistic Time`?

**Review by ic4y — DISMISSED** [review]
Nice

**Comment by ic4y** [comment]
> 
Either `Last Statistic Time` or `Last Stats Time` is fine

**Comment by ic4y** [comment]
> > > image
> > 
> > 
> > Is `Last Run Time` called `Last Print Time` or something else?
> 
> Good idea. How about `Last Statistic Time`?

Either `Last Statistic Time` or `Last Stats Time` is fine.

File: seatunnel-engine/seatunnel-engine-client/src/main/java/org/apache/seatunnel/engine/client/job/JobMetricsRunner.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+@Slf4j
+public class JobMetricsRunner extends Thread {
```
Comment (hailin0) [inline seatunnel-engine/seatunnel-engine-client/src/main/java/org/apache/seatunnel/engine/client/job/JobMetricsRunner.java]:
```suggestion
public class JobMetricsRunner implements Runnable {
```

Comment (TyrantLucifer) [reply seatunnel-engine/seatunnel-engine-client/src/main/java/org/apache/seatunnel/engine/client/job/JobMetricsRunner.java]: Done, PTAL

File: seatunnel-core/seatunnel-starter/src/main/java/org/apache/seatunnel/core/starter/seatunnel/command/ClientExecuteCommand.java [hunk 2]
```diff
@@ -89,9 +101,19 @@ public void execute() throws CommandExecuteException {
...
-                clientJobProxy.waitForJobComplete();
+                // get job id
                 long jobId = clientJobProxy.getJobId();
-                System.out.println(engineClient.getJobMetrics(jobId));
+                JobMetricsRunner jobMetricsRunner = new JobMetricsRunner(engineClient, jobId);
+                executorService = Executors.newSingleThreadScheduledExecutor();
+                executorService.scheduleAtFixedRate(jobMetricsRunner, 0,
+                        seaTunnelConfig.getEngineConfig().getPrintJobMetricsInfoInterval(), TimeUnit.SECONDS);
+                // get job start time
+                startTime = LocalDateTime.now();
```
Comment (EricJoy2048) [inline seatunnel-core/seatunnel-starter/src/main/java/org/apache/seatunnel/core/starter/seatunnel/command/ClientExecuteCommand.java]: Move `startTime = LocalDateTime.now();` before `ClientJobProxy clientJobProxy = jobExecutionEnv.execute();` ？

Comment (TyrantLucifer) [reply seatunnel-core/seatunnel-starter/src/main/java/org/apache/seatunnel/core/starter/seatunnel/command/ClientExecuteCommand.java]: Done

**Review by EricJoy2048 — APPROVED** [review]

**Review by Hisoka-X — APPROVED** [review]

**Pull request merged**